src/Main.elm

```diff
@@ -59,7 +55,7 @@ type Msg
     | PageLinkClicked Browser.UrlRequest
     | UrlChanged Url.Url
     | DefinitionMsg Definition.Msg
-    | HelpSelfMsg HelpSelf.Msg
+    | HelpSelfSingleMsg HelpSelfSingle.Msg
```

Comment:
Does this need HelpSelfGridMsg HelpSelfGrid.Msg?

Reply:
Right now the grid page doesn't have any Msg types